src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq.c

```diff
@@ -78,6 +78,10 @@
 static DEFINE_IDR(xocl_xgq_vmr_cid_idr);
 #define XOCL_VMR_INVALID_CID	0xFFFF
 
+/* Retry is set to 200 seconds for SC to be active*/
+#define MAX_SC_WAIT             200 //sec
```

Comment:
Please refactor the code to have this value actually bee 200 seconds and NOT 200 loops.

Once should be able to change the SC_WAIT_INTERVAL without needing to change this value.